Meta/Lagom/Demos/DemoGUI.cpp

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (c) 2022, Filiph Sandström <filiph.sandstrom@filfatstudios.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <LibCore/System.h>
+#include <LibGUI/AboutDialog.h>
+#include <LibGUI/Application.h>
+#include <LibGUI/BoxLayout.h>
+#include <LibGUI/Button.h>
+#include <LibGUI/Label.h>
+#include <LibGUI/MessageBox.h>
+#include <LibGUI/Window.h>
+#include <LibGfx/FontDatabase.h>
+#include <LibGfx/Palette.h>
+#include <LibGfx/SystemTheme.h>
+#include <LibMain/Main.h>
+
+ErrorOr<int> serenity_main(Main::Arguments arguments)
+{
+    // Set default font path.
+    // TODO: figure out a better way to do this :^)
+    Gfx::FontDatabase::set_default_fonts_lookup_path("../../Base/res/fonts");
```

Comment:
Did you try this suggestion from Discord? Basically passing `SERENITY_SOURCE_DIR` to the executable from CMakeLists.txt:
https://discord.com/channels/830522505605283862/830522505605283866/943269707019067392

Reply:
@trflynn89 yeah I did, mostly left it like that since it’s a demo and I want to be explicit on how to set it up properly without requiring knowledge of our cmake setup. I can change it if your prefer (and add a long comment or something to explain why and how this needs to be done atm).

Reply:
Let's just check `SERENITY_SOURCE_DIR` at the top of `serenity_main` here. Most of us use `serenity.sh` which will set that env if it isn't set.
```diff
diff --git a/Meta/Lagom/Demos/DemoGUI.cpp b/Meta/Lagom/Demos/DemoGUI.cpp
index 8feb77098e..f2ccf954cc 100644
--- a/Meta/Lagom/Demos/DemoGUI.cpp
+++ b/Meta/Lagom/Demos/DemoGUI.cpp
@@ -16,16 +16,23 @@
 #include <LibGfx/Palette.h>
 #include <LibGfx/SystemTheme.h>
 #include <LibMain/Main.h>
+#include <stdlib.h>
 
 ErrorOr<int> serenity_main(Main::Arguments arguments)
 {
+    char const* serenity_source_dir = getenv("SERENITY_SOURCE_DIR");
+    if (serenity_source_dir == nullptr) {
+        // Assume running from Build/lagom.
+        serenity_source_dir = "../..";
+    }
+
     // Set default font path.
     // TODO: figure out a better way to do this :^)
-    Gfx::FontDatabase::set_default_fonts_lookup_path("../../Base/res/fonts");
+    Gfx::FontDatabase::set_default_fonts_lookup_path(String::formatted("{}/Base/res/fonts", serenity_source_dir));
     // Set the default font, this is required here since we're not running in a Serenity env
     Gfx::FontDatabase::set_default_font_query("Katica 10 400 0"sv);
     // Ditto but for the theme and palette
-    Gfx::set_system_theme(Gfx::load_system_theme("../../Base/res/themes/Default.ini"));
+    Gfx::set_system_theme(Gfx::load_system_theme(String::formatted("{}/Base/res/themes/Default.ini", serenity_source_dir)));
 
     auto app = TRY(GUI::Application::try_create(arguments));
     app->set_system_palette(Gfx::current_system_theme_buffer());
```

Reply:
@trflynn89 that worked perfectly!